src/NavWrapper/NavWrapper.tsx

```diff
@@ -151,7 +150,7 @@ export const NavWrapper: React.FC<NavWrapperProps> = () => {
                                         setSoftReloadFlag(true);
                                     })();
                                 }}>
-                                    {session.userType === UserRole.STUDENT ? 'Professor' : 'Student'} View (BETA)
```

Comment:
I'ono, we just made some changes to routing. I personally think the `Beta` is a positive thing to have in the application, rather than something we should rush to remove.